[rospy.client][INFO] 2020-11-24 03:45:27,363: init_node, name[/train_3240_1606189527331], pid[3240]
[xmlrpc][INFO] 2020-11-24 03:45:27,364: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2020-11-24 03:45:27,365: Started XML-RPC server [http://192.168.0.191:44915/]
[rospy.init][INFO] 2020-11-24 03:45:27,365: ROS Slave URI: [http://192.168.0.191:44915/]
[rospy.impl.masterslave][INFO] 2020-11-24 03:45:27,365: _ready: http://192.168.0.191:44915/
[rospy.registration][INFO] 2020-11-24 03:45:27,367: Registering with master node http://192.168.0.201:11311
[xmlrpc][INFO] 2020-11-24 03:45:27,367: xml rpc node: starting XML-RPC server
[rospy.init][INFO] 2020-11-24 03:45:27,466: registered with master
[rospy.rosout][INFO] 2020-11-24 03:45:27,466: initializing /rosout core topic
[rospy.rosout][INFO] 2020-11-24 03:45:27,472: connected to core topic /rosout
[rospy.simtime][INFO] 2020-11-24 03:45:27,473: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rospy.internal][INFO] 2020-11-24 03:45:27,494: topic[/bev/flow_image] adding connection to [http://192.168.0.201:45935/], count 0
[rospy.internal][INFO] 2020-11-24 03:45:27,501: topic[/scan] adding connection to [http://192.168.0.201:41579/], count 0
[rospy.internal][INFO] 2020-11-24 03:45:27,505: topic[/scan] adding connection to [http://192.168.0.201:38893/], count 1
[rospy.internal][INFO] 2020-11-24 03:45:27,510: topic[/odom] adding connection to [http://192.168.0.201:41579/], count 0
[rospy.internal][INFO] 2020-11-24 03:45:27,511: topic[/start_goal_points] adding connection to [http://192.168.0.201:40621/], count 0
[rospy.internal][INFO] 2020-11-24 03:45:27,514: topic[/bev/occupancy_image] adding connection to [http://192.168.0.201:45935/], count 0
[rospy.internal][INFO] 2020-11-24 03:45:27,630: topic[/cmd_vel] adding connection to [/gazebo], count 0
[rospy.internal][INFO] 2020-11-24 03:45:27,631: topic[/cmd_vel] adding connection to [/bev_flow_estimator], count 1
[rospy.internal][INFO] 2020-11-24 03:45:27,729: topic[/rosout] adding connection to [/rosout], count 0
[rospy.internal][INFO] 2020-11-24 03:45:27,730: topic[/is_created_scenarios] adding connection to [/episode_manager], count 0
[rospy.core][INFO] 2020-11-24 03:45:30,139: signal_shutdown [atexit]
[rospy.internal][INFO] 2020-11-24 03:45:30,143: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2020-11-24 03:45:30,143: topic[/cmd_vel] removing connection to /gazebo
[rospy.internal][INFO] 2020-11-24 03:45:30,143: topic[/cmd_vel] removing connection to /bev_flow_estimator
[rospy.internal][INFO] 2020-11-24 03:45:30,144: topic[/is_created_scenarios] removing connection to /episode_manager
[rospy.internal][INFO] 2020-11-24 03:45:30,144: topic[/bev/flow_image] removing connection to http://192.168.0.201:45935/
[rospy.internal][INFO] 2020-11-24 03:45:30,144: topic[/bev/occupancy_image] removing connection to http://192.168.0.201:45935/
[rospy.internal][INFO] 2020-11-24 03:45:30,144: topic[/odom] removing connection to http://192.168.0.201:41579/
[rospy.internal][INFO] 2020-11-24 03:45:30,145: topic[/start_goal_points] removing connection to http://192.168.0.201:40621/
[rospy.internal][INFO] 2020-11-24 03:45:30,145: topic[/scan] removing connection to http://192.168.0.201:41579/
[rospy.internal][INFO] 2020-11-24 03:45:30,146: topic[/scan] removing connection to http://192.168.0.201:38893/
[rospy.impl.masterslave][INFO] 2020-11-24 03:45:30,146: atexit
[rospy.internal][WARNING] 2020-11-24 03:45:30,147: Unknown error initiating TCP/IP socket to 192.168.0.201:58309 (http://192.168.0.201:41579/): Traceback (most recent call last):
  File "/opt/ros/noetic/lib/python3/dist-packages/rospy/impl/tcpros_base.py", line 562, in connect
    self.read_header()
  File "/opt/ros/noetic/lib/python3/dist-packages/rospy/impl/tcpros_base.py", line 657, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
AttributeError: 'NoneType' object has no attribute 'buff_size'

[rospy.internal][WARNING] 2020-11-24 03:45:30,147: Unknown error initiating TCP/IP socket to 192.168.0.201:57717 (http://192.168.0.201:40621/): Traceback (most recent call last):
  File "/opt/ros/noetic/lib/python3/dist-packages/rospy/impl/tcpros_base.py", line 562, in connect
    self.read_header()
  File "/opt/ros/noetic/lib/python3/dist-packages/rospy/impl/tcpros_base.py", line 657, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
AttributeError: 'NoneType' object has no attribute 'buff_size'

